src/lib/Libtpp/tpp_em.c

```diff
@@ -1086,22 +1077,7 @@ tpp_em_wait_win(void *em_ctx, em_event_t **ev_array, int timeout)
 		ptv = &tv;
 	}
 
-	DBPRT(("## tpp_em_wait_win: ctx->maxfd=%d\n", ctx->maxfd));
-
-	for(i = 0; i < ctx->maxfd; i++){
-		if(FD_ISSET(i, &ctx->read_fds) || FD_ISSET(i, &ctx->write_fds) || FD_ISSET(i, &ctx->err_fds))
-			flag_fd_isset = 1;
-		if(FD_ISSET(i, &ctx->read_fds))
-			DBPRT(("## tpp_em_wait_win: fd=%d is set for read\n", i));
-		if(FD_ISSET(i, &ctx->write_fds))
-			DBPRT(("## tpp_em_wait_win: fd=%d is set for write\n", i));
-		if(FD_ISSET(i, &ctx->err_fds))
-			DBPRT(("## tpp_em_wait_win: fd=%d is set for err\n", i));
-	}
-	if(!flag_fd_isset)
-		return 0;
```

Comment:
You should really squash and rebase for before initiating your review, since these changes are not existent on the codeline.

Reply:
Will keep that in mind.